vispy/testing/_runners.py

```diff
@@ -190,8 +190,15 @@ def _check_line_endings():
 """
 
 
-def _examples():
-    """Run all examples and make sure they work
+def _examples(fnames_str):
+    """Run examples and make sure they work.
+
+    Parameters
+    ----------
+
```

Comment:
Name is fine by me, but under `numpy` docstring spec this newline also shouldn't be there :)